packages/react/src/components/BarChartCard/BarChartCard.jsx

```diff
@@ -214,7 +214,7 @@ const BarChartCard = ({
   const options = useMemo(
     () => ({
       animations: false,
-      accessibility: true,
+      accessibility: false,
```

Comment:
what does this do?

Reply:
Unintentional change, but also I'm not sure what actual effect it has on the underlying ComboChart carbon-charts control. The TimeSeries card uses `accessibility: false`, the BarChartCard uses `accessibility: true`

Reply:
Oof. Might need to figure out which one we should be using